### ISSUE TITLE
Skin lesion with central keratin plug

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9997]
**Preferred term label:**
Skin lesion with central keratin plug

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Keratoacanthoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
should the label of this term be Keratoacanthoma ?

**Comment by MickeySegal** [comment]
The term we had was a clinical description but had a pathological observation.  Since we only uses this finding for the disease Keratoacanthoma I will use the pre-existing https://hpo.jax.org/app/browse/term/HP:0031525

**Comment by pnrobinson** [comment]
OK!
